src/current/cockroachcloud/folders.md

```diff
@@ -5,27 +5,25 @@ toc: true
 docs_area: manage
 ---
 
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters with folders using the CockroachDB {{ site.data.products.cloud }} Console. You can also use the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}), or the [Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) v1.1.0 or above. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}]({% link cockroachcloud/managing-access.md %}).
+
 {{site.data.alerts.callout_success}}
-{% include_cached feature-phases/limited-access.md %}
+{% include_cached feature-phases/preview.md %}
 {{site.data.alerts.end}}
 
-This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters with folders using the CockroachDB {{ site.data.products.cloud }} Console. You can also use the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}), or the [Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) v1.1.0 or above. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}]({% link cockroachcloud/managing-access.md %}).
-
 ## How folders work
 
 Folders allow you to organize and manage access to your clusters according to your organization's requirements. For example, you can create top-level folders for each business unit in your organization, and within those folders, you can organize clusters by geographic location and then by their level of maturity, such as production, staging, and testing. For more details, refer to [Folder Structure](#folder-structure).
 
 Each folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
 
-Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID.
+Each folder that you create has an ID, which determines its position in the organization's hierarchy.
 
-If no parent ID is specified for a cluster, it is created at the root level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
+- Clusters or folders within a folder have their parent ID set to that folder's ID.
+- If a cluster or folder has no parent ID, or if its parent ID is set to `root`, its location is the root level of the organization.
```

Comment:
I might word this differently but its up to you.  maybe something more declarative like:

Clusters and folders residing at the root level of the organization have no parent ID.  When using the API, a value of "root" is also accepted as a convenience when creating folders.

The current form makes it sound like a folder or cluster without a parent might have no parent ID or it might have a parent ID which is "root" which isn't corrently.

Reply:
Discussed in Slack and we came up with something reasonable. :)